client/src/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { Fragment } from 'react';
+import { Provider } from 'react-redux';
+import store from './store';
+import Landing from './components/layout/Landing';
+import './App.css';
```

Comment:
my js isn't 🔥 yet. What exactly are all of these imports doing?

Reply:
Some of these are UI components (`Fragment`, `Landing`) and some are to do with redux for state management (`Provider`, `store`)